core/object/object.h

```diff
@@ -39,6 +39,7 @@
 #include "core/templates/hash_set.h"
 #include "core/templates/list.h"
 #include "core/templates/safe_refcount.h"
+#include "core/variant/required_ptr.h"
```

Comment:
Is this strictly necessary to include in `object.h`, it'll make this header cause mass rebuilds when changed